web/react/components/user_settings/user_settings_notifications.jsx

```diff
@@ -294,7 +294,7 @@ class NotificationsTab extends React.Component {
                 <span>
                     <FormattedMessage
                         id='user.settings.notifications.info'
-                        defaultMessage='Desktop notification sounds are available on Firefox, Safari, Chrome, Internet Explorer, and Edge.'
+                        defaultMessage='Desktop notification are available on Firefox, Safari, Chrome, Internet Explorer, and Edge.'
```

Comment:
Minor typo here. You have "Desktop notification are" instead of "Desktop notifications are"